### DISCUSSION TITLE
"expand all" roadmap page button

### DISCUSSION THREAD

**Discussion by vankhoawin** [twilio-labs/paste#1546] [Ideas] [answered]
Hello team! this is a very, very minor quality of life improvement, but on the https://paste.twilio.design/roadmap/ i frequently open up all of the disclosures to search for a component that's coming up in the future. it would be nice if they were expanded by default or if there were some kind of "expand/minimize all" button. FWIW, i'm not sure if the amount of text ever bothered me enough for me to minimize any release sections.

**Comment by richbachman** [comment, marked as answer]
Hi @vnguyen94. Good suggestion. That page really needs a rework. I've shared the feedback with the team, and we'll take it into consideration when we rework that page.

**Reply by vankhoawin** [reply]
Awesome, thanks Rich 👍
